docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -358,6 +380,14 @@ In order to add a master, the Alluxio cluster must be in HA mode. If you are run
 a single master cluster, you must configure it to be an HA cluster to be able to have more than 1
 master.
 
+#### Alluxio HA cluster with internal leader election
```

Comment:
embedded journal doesn't support changing the set of masters

Reply:
Can't the users modify the properties and restart the whole cluster when using the embedded journal.

Reply:
Only if they re-format. When the journal is initialized, it records which hosts are members of the cluster. This list is stored on disk, and we don't currently provide a way to change it. The only way to change masters is to shut down the cluster, format the journal, update the configuration, then start

Reply:
Changed to `When internal leader election is used, Alluxio masters are determined. Adding or Removing master nodes
is not supported. `, thanks

Reply:
It's possible to change masters, with the major caveat that they need to re-format their journal

Reply:
is it safe to only format the master journal without formating the AlluxioWorkers?

Reply:
whenever masters are formatted, workers should also be formatted. Otherwise they will have orphaned blocks on their ramdisks

Reply:
orphaned blocks don't cause any errors, but they waste space

Reply:
Got it, it's the same as erased the whole cluster and build a whole new cluster. Thanks